src/vtk_calculator.py

```diff
@@ -1,53 +1,106 @@
 #!/usr/bin/env python3
 """Evaluates a function on a given mesh, using the VTK Calculator."""
 
-import argparse, logging, os.path
-import mesh_io
+import argparse
+import logging
+import os.path
 import vtk
+import json
+import numpy as np
 
 
 def parse_args():
-    parser = argparse.ArgumentParser(description = __doc__)
+    parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument("in_meshname", metavar="inputmesh", help="The mesh used as input")
-    parser.add_argument("function", 
-            help="""The function to evalutate on the mesh. 
-            Syntax is the same as used in the calculator object, coordinates are given as e.g. 'coordX'.""")
-    parser.add_argument("--out", "-o", dest="out_meshname", help="""The output meshname. 
+    parser.add_argument("function",
+                        help="""The function to evalutate on the mesh.
+            Syntax is the same as used in the calculator object, coordinates are given as e.g. 'coordsX+coordsY'.""")
```

Comment:
This change breaks all existing scripts.
Please use `x, y, z` as names for the inputs.
Also, this should still support python functions from the `math` package. You could simply remove all mentions of `math.` in the function.

Does this support functions such as `exp()`, `sin()` etc?

Reply:
Supported operators cover most (all?) of the python `math` package.  I changed names to `x,y,z`.
`+ - * / ^`

` abs
 acos
 asin
 atan
 ceil
 cos
 cosh
 exp
 floor
 log
 mag
 min
 max
 norm
 sign
 sin
 sinh
 sqrt
 tan
 tanh`

Reply:
This should be fine. Allowing `math.` is not that important.